frontends/pytorch/csrc/builder/module_builder.cpp

```diff
@@ -95,6 +95,28 @@ ModuleBuilder::startCaptureFunction(std::string &name,
   return std::make_shared<AcapController>(typeMapper, std::move(funcBuilder));
 }
 
+void ModuleBuilder::importFunction(torch::jit::StrongFunctionPtr function) {
+  auto inserter = createInserter();
```

Comment:
should we be looking at `function.cu_`? If not, maybe leave a comment. (perhaps about why we take a StrongFunctionPtr instead of a Function directly).

Reply:
What we actually want is a torch.jit.ScriptFunction, but there is naming cruft here on the PyTorch<->Pybind bridge side. Left a comment.

  // Imports a traced function. Note that the python type
  // torch.jit.ScriptFunction is the C++ type torch::jit::StrongFunctionPtr.
  // Just a bit of naming cruft.